tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/LangLibFunctionTest.java

```diff
@@ -338,11 +338,11 @@ public void testIntersectionType() {
         assertEquals(type.typeKind(), INTERSECTION);
 
         List<String> expFunctions = List.of("reduce", "forEach", "shift", "length", "sort", "reverse", "toStream",
-                                            "remove", "push", "filter", "pop", "lastIndexOf", "iterator", "removeAll",
-                                            "setLength", "slice", "enumerate", "unshift", "map", "indexOf",
-                                            "cloneWithType", "cloneReadOnly", "toBalString", "toJson", "isReadOnly",
-                                            "fromJsonWithType", "mergeJson", "clone", "ensureType", "toString",
-                                            "toJsonString");
+                "remove", "push", "filter", "pop", "lastIndexOf", "iterator", "removeAll",
+                "setLength", "slice", "enumerate", "unshift", "map", "indexOf",
+                "cloneWithType", "cloneReadOnly", "toBalString", "toJson", "isReadOnly",
+                "fromJsonWithType", "mergeJson", "clone", "ensureType", "toString",
+                "toJsonString");
```

Comment:
This file seems to  contain a lot of formatting changes? Better if the IDE is set to format only the modified sections of a file